docs/source/get_involved/code_of_conduct.rst

```diff
@@ -342,14 +397,161 @@ Guide <https://www.djangoproject.com/conduct/>`__ and we are also
 grateful for `this guidance from Ada
 Initiative <http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Responding_to_reports>`__.
 
-We really appreciate the work that all of the communities linked above
-have put into creating such a well considered process.
+We highly appreciate the work that all of the communities linked above
+have put into creating code of conduct documents and processes.
 
 This Code of Conduct is licensed under a `Creative Commons Attribution
 4.0 International <https://creativecommons.org/licenses/by/4.0/>`__ (CC
 BY 4.0 CA) license which means you are free to share and adapt the work
-so long as the attribution to `The Turing
-Way <https://github.com/alan-turing-institute/the-turing-way>`__ project
-with Dr Kirstie Whitaker as the principal investigator is retained,
-along with the attribution to the Carpentries, The Alan Turing Institute
-Data Study Group organising team, Citizen Lab and the other resources.
+so long as attribution is maintained to substantial sources of contribution:
+
+* The `Django project <https://www.djangoproject.com/>`__
+* The `Contributor Covenant <http://contributor-covenant.org/>`__,
+* The `Citizen Lab <https://citizenlab.ca/>`__
+* The `Alan Turing Institute Data Study Groups <https://www.turing.ac.uk/collaborate-turing/data-study-groups>`__ organising team
+* `The Turing Way <https://github.com/alan-turing-institute/the-turing-way>`__ project
+* `The Carpentries <https://carpentries.org/>`__
+* The ``sktime`` project
+* other sources and influences mentioned above
+
+
+Appendix A: Guidance in applying the CoC
+----------------------------------------
+
+This appendix provides some additional guidance in applying the CoC.
+It contains sections on:
+
+* how to apply the CoC in some common situations where principles might conflict
+* investigating CoC violations in resource allocation
+
+A.1 Resolving conflicting CoC principles
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sometimes, CoC requirements end up in a trade-off, or in conflict with each other.
+We outline a few guiding examples in how these should be traded off.
+
+Examples:
+
+* contributor criticizing leadership for handing resources to personal network, leadership invoking "discomfort"
+* passionate disagreement on technical content between contributors
+* accidental violations of CoC causally due to protected characteristics of the person violating the CoC
+* systemic discrimination issues whose resolution would cause discomfort in the community
+
+Guidance rules:
+
+* the less powerful party should be protected, i.e., CoC cannot be invoked by lead community member to silence critique of how funds are spent.
+  But, CoC should protect new contributors or early career members from more powerful community members, e.g., in operational or technical disputes.
```

Comment:
it means:

* they should try convince the core developers, say, by producing evidence that the process was fair. The burden of proof should be with leadership, to show the process was fair.
* the CoC will not move to silence the early career member in any way, as long as the allegations are plausible and civil in tone.
* in particular, leadership just has to stomach the accusations as long as they remain even vaguely plausible. It's a moment of accountability.

Reply:
Strongly disagree with "vaguely plausible". By proof I don't mean that the person suspecting favouritism has to provide the videos/messages of this leader discussing how they're going to hire their favourite. Just stating the facts that, for example, this leader had candidate A that has the same level of competence and is ready to start the project at the same time as B but chooses B, and B happens to be his nephew - is a proof and it must be provided by the person complaining to the CoCC. This person can not accuse leadership of favouritism just based on their impressions or feelings, zero facts.

Reply:
> This person can not accuse leadership of favouritism just based on their impressions or feelings, zero facts.

I think the standard should be, "plausible suspicion".
The plausible suspicion must be outlined to the CoC committee for an investigation.

If the community member decides to make it public in the community based on plausible suspicion, I think that should be ok, in the interest of accountability, checks and balances.

Similar to how it is fine for free media to allege wrongdoing of public role holders on a basis of plausibility - in countries that allow freedom of speech, the rules are less stringent in protecting public persons because of the public interest and the accountability moment.

Reply:
In particular, I would *strongly disagree* that the community member must be silent until the CoC committee has completed their investigation, say.

They only need to be civil in tone, but still can present the evidence basis for their suspicion.

This should *not* be stifled or silenced by invocation of things like "individual discomfort" (of the person in power).